libraries/AP_Math/SCurve.cpp

```diff
@@ -71,7 +72,7 @@ void SCurve::calculate_track(const Vector3f &origin, const Vector3f &destination
     jerk_max = jerk_maximum;
 
     // update speed and acceleration limits along path
-    set_kinematic_limits(origin, destination,
+    set_kinematic_limits(direction,
```

Comment:
We call this vector "track" as it has both direction and length. A direction vector would be unit length.

Reply:
Makes sense!

I renamed it to track_curr and track_temp to reflect that. I can't use `track` for local members as that currently is used as a private member in the class.